config/initializers/assets.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.config.assets.version = "1.0"
 
 Rails.application.config.assets.paths << Rails.root.join("node_modules")
+
+Rails.application.config.assets.precompile << "login.css"
```

Comment:
Không được làm như này nhé, chỉ nên có 1 entry point duy nhất cho toàn bộ các file scss, các file scss/css khác sẽ được import vào trong file đó, chứ không thêm precompile từng file riêng lẻ như này

Reply:
Okee bạn nha